scanners/git-repo-scanner/templates/git-repo-scanner-scan-type.yaml

```diff
@@ -20,7 +20,8 @@ spec:
               image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.Version }}"
               command:
```

Comment:
We might have to set the work dir here too.

Reply:
This is fine, because we already set it in the dockerfile.